### ISSUE TITLE
Worked for me without redefining x, y, and z

### ISSUE THREAD

**Issue by mackenzieyoung** [abbynormative/dvp-u3-constellations#1]
https://github.com/abbynormative/dvp-u3-constellations/blob/036152511740ec754c499d199880e52a889ab03a/orion-constellation/orion_constellation.py#L106-L109

I wonder why this wasn't working for you -- when I commented out this part it still worked fine for me both when I ran the .py file on my computer and when I ran the jupyter notebook file. Are you sure you ran all the previous cells before running this one?

**Comment by abbynormative** [comment]
Yep, I did run all previous cells. Do you want me to resubmit without those
extra lines?

**Comment by mackenzieyoung** [comment]
No it's fine! I wonder why it was giving you that error...
